skt/misc.py

```diff
@@ -11,9 +11,95 @@
 # You should have received a copy of the GNU General Public License
 # along with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+"""Functions used by multiple parts of skt."""
```

Comment:
If you want to add a docstring, can it read "Functions and constants", since, well, it's not only functions in the module?